posts/2017-06-27-announcing-emit-2017.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Announcing Emit 2017
+description: 'Announcing Emit, the conference on event-driven arcitectures.'
+date: 2016-06-27
+layout: Post
+authors:
+  - NickGottlieb
+---
+
+# Announcing Emit 2017 - the conferences on event-driven architectures 
+
+<img src="https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/emit_logo+(1).png">
+
+## Why
+The serverless movement is all about productivity, speed, and results. These are the values that make serverless powerful and that the community has rallied around. As the creators and maintainers of the Serverless Framework, we’ve enjoyed a front-row seat to watch the serverless movement grow and evolve. What started as a small group of passionate, forward-thinking developers scratching their own itch has evolved in to Fortune 500 companies rethinking the way that they build and architect their applications in order to empower their development teams to get to market faster.
+
+One theme that we’ve seen a growing amount of interest in is the event-driven nature of serverless architectures. While event-driven architectures are certainly nothing new, the emergence of serverless compute has made them far more powerful, relevant, and exciting. The growing interest and excitement around this theme is what prompted us to create Emit, a conference focused on event-driven architectures.
+
+## What
+Emit will be a single-day, single-track event with technical thought leaders in the space sharing knowledge, experiences, and predictions around event-driven architectures. It will be small, intimate, and more like a summit than a traditional technology conference (i.e no sponsor booths). 
+
+There will be ample networking opportunities as well as a closing reception during which discussion, debate, and free-flowing ideas will be encouraged. 
+
+## When/Where
+Emit will take place on August 17th in San Francisco at [The Pearl](http://thepearlsf.com/). The Pearl is a beautiful space with a large auditorium for the speakers and panel, a mezzanine which will serve as a lounge for networking and relaxing, and well as a large rooftop where lunch and the closing reception will be held. 
+
+
+## Speakers
+We have an extremely exciting group of speakers lined up, made up of leading thinkers in the serverless and event-driven space, including technology leaders from organizations such as Nordstrom, CapitalOne, Uber, Coca-Cola, IBM, Google, Microsoft, and more. We'll be adding more speakers over the coming weeks but an up to date list can be found on [emitconference.com](http://www.emitconference.com/).
```

Comment:
Should we add links to `Nordstrom`, `CapitalOne` etc. here? Not 100% sure but I heard that this could also be beneficial for SEO.